### PR TITLE
Fixes #24201: rudder-pkg stops at first install error on the upgrade-all command

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/24201

### CODE REVIEW

File: relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py [hunk 1]
```diff
@@ -98,6 +100,7 @@ def run(
             "Command '%s' returned a non-zero return code, %d.%s%s"
             % (' '.join(e.cmd), e.returncode, stdout, stderr),
             1,
+            exit_on_error
```
Comment (peckpeck) [inline relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py:103]: why is the previous call using an if and this one using an additional parameter ?

**Comment by Fdall** [comment]
Commit modified

**Review by ncharles — APPROVED** [review]
let's accept this to see what is broken

**Comment by VinceMacBuche** [comment]
tests are not ok due to not correct formatting in rudder-pkg

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**